### PR TITLE
add independent ftl

### PR DESCRIPTION
**Pull request by Errolyn**
Converting independent over to fluent

bedrock pr: https://github.com/mozilla/bedrock/pull/9105

### CODE REVIEW

File: en/firefox/features/independent.ftl [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+features-independent-firefox-is-independent = { -brand-name-firefox } is independent and a part of the non-profit { -brand-name-mozilla }, which fights for your online rights, keeps corporate powers in check and makes the Internet accessible to everyone, everywhere.
+
+# This phrase means without conditions or restrictions, expecting nothing in return
+features-independent-no-strings-attached = No strings attached
+features-independent-firefox-is-built-by = { -brand-name-firefox } is built by a non-profit. That means we can do things that others can’t, like build new products and features without a hidden agenda. We champion your right to privacy with tools like Private Browsing with Tracking Protection.
+
+# Obsolete string
+features-independent-firefox-is-built-by-old = { -brand-name-firefox } is built by a non-profit. That means we can do things that others can’t, like build new products and features without a hidden agenda. We champion your right to privacy with tools like Private Browsing with Tracking Protection, which go beyond what { -brand-name-google } { -brand-name-chrome } and { -brand-name-microsoft } { -brand-name-edge } offer.
```
Comment (peiying2) [inline en/firefox/features/independent.ftl:20]: Let's standardize how we distinguish the obsolete and the updated string in the string ID. I have seen `old`, `fallback`, and `update`. The first two label the obsolete strings while the latter marks the revised string.

**Comment by Errolyn** [comment]
@peiying2 

Hmm this seems like something that might need a style guide or equivalent, or it's going to end up being different in every PR. both Craig and I worked on doing the feature page string conversions, and our styles of how to mark them ended up being different. 

What would you prefer? Should the string variable names even have "old" or "obsolete" in them? maybe they should only be in a comment since when we update strings in the future we wont be able to update the old string variable to have "old" in the name?

I have no preference here so I think which ever way you would like to see them come through works for me.

**Comment by Pike** [comment]
Recommended strings  IDs and revisioning them is documented in https://bedrock.readthedocs.io/en/latest/l10n.html#ftl-string-ids

**Comment by peiying2** [comment]
> Recommended strings IDs and revisioning them is documented in https://bedrock.readthedocs.io/en/latest/l10n.html#ftl-string-ids

@Errolyn you can add comments to the old string as obsolete, but in the string ID for the newer version, let's use xxx-v2 for the updated string.

**Comment by craigcook** [comment]
I didn't realize there was already a convention established. I don't think we've put it into practice anywhere yet, but I guess we can start now 😃.

What should we do when a string is replaced by a new string with an entirely different ID (because it has entirely different text)? Does the new one still have a `-v2` even when the rest of the ID is different? And is it still useful to mark old ones with `# obsolete`?

**Comment by Errolyn** [comment]
It kinda seems like we need to go back through and do that to the 4ish prs we had merged earlier then too?

**Comment by peiying2** [comment]
> I didn't realize there was already a convention established. I don't think we've put it into practice anywhere yet, but I guess we can start now 😃.
> 
> What should we do when a string is replaced by a new string with an entirely different ID (because it has entirely different text)? Does the new one still have a `-v2` even when the rest of the ID is different? And is it still useful to mark old ones with `# obsolete`?

v2 is only for identical string IDs when a string is updated, so it can distinguish the two. For completely new IDs because the new string is very different from the old one, let's **not** use v2. The new ID is doing that. 

Now, if the new content needs to be switched on by certain date for a particular product launch, is v2 easier to track that or #obsolete comment does the job when you want to switch over by a defined date regardless the revised strings are localized or not

**Comment by Errolyn** [comment]
Ok I have remove -old from the obsolete string and add v2 to the new string which has the same beginning but a different ending.

**Pull request merged**